src/Mjolnir.ts

```diff
@@ -148,13 +149,15 @@ export class Mjolnir {
         }
         await logMessage(LogLevel.INFO, "index", "Mjolnir is starting up. Use !mjolnir to query status.");
 
-        return new Mjolnir(client, protectedRooms, banLists);
+        const ruleserver = config.web.ruleserver ? new RuleServer() : null;
+        return new Mjolnir(client, protectedRooms, banLists, ruleserver);
```

Comment:
Note to self: I think that we're starting to reach the stage at which we should pass an object to the constructor of `Mjolnir` rather than a list of arguments. This can wait for a followup, though.